src/batch/src/executor/filter.rs

```diff
@@ -17,7 +17,7 @@ use risingwave_common::array::ArrayImpl::Bool;
 use risingwave_common::array::DataChunk;
 use risingwave_common::catalog::Schema;
 use risingwave_common::util::chunk_coalesce::DataChunkBuilder;
-use risingwave_expr::expr::{build_from_prost, BoxedExpression};
+use risingwave_expr::expr::{build_batch_expr_from_prost, BoxedExpression};
```

Comment:
Does it mean that `build_from_prost` will not be directly used anymore?

Reply:
It is still used quite a bit. Only `Filter` and `Project` have been changed to `build_batch_expr_from_prost`. Examples of the rest that still uses `build_from_prost` include

- `HopWindowExecutor`: because the expression is derived from +/- interval, so it must be non-fallable
- `SortAggExecutor`: same as above, but it's even simpler - there are only `ValueRef`s
- `UpdateExecutor`: because I can't image why users need to ignore expressions here, so I just keep it unchanged.